src/me.rs

```diff
@@ -989,19 +1448,19 @@ fn get_subpel_mv_rd_cost<T: Pixel>(
     cand_mv,
   );
   let plane_ref = tmp_region.as_const();
-  compute_mv_rd_cost(
+  compute_mv_rd(
     fi, pmv, lambda, use_satd, bit_depth, bsize, cand_mv, org_region,
     &plane_ref,
   )
-  .0
 }
 
+/// Compute the rate distortion stats for a motion vector.
 #[inline(always)]
-fn compute_mv_rd_cost<T: Pixel>(
+fn compute_mv_rd<T: Pixel>(
```

Comment:
This is a nice cleanup. Was it needed for UMH or just convenient?

Reply:
I wanted to consolidate the rd information into a struct, so additional motion search modes would be easier to read. It basically came down to that.